### ISSUE TITLE
Loss 0 during Stage 3 Training? + Stage 2 Model

### ISSUE THREAD

**Issue by jamespark3922** [CircleRadon/Osprey#13]
Hi I was able to train the stage 2 model and moving onto stage 3 training.

Here is the command I am running. The loss seems fine if I change the `model_name_or_path` to the vicuna lm model instead of stage 2 trained model. Do you see any issue, and is it possible to share the stage 2 model as well? 
```
deepspeed --include localhost:0,1  osprey/train/train_mem.py \
    --deepspeed ./scripts/zero2.json \
    --model_name_or_path exp/stage2_slurm_no_grad_ckpt/ \
    --dataset_config ./osprey/configs/stage3.json \
    --version v1 \
    --vision_tower $MODELS/CLIP-convnext_large_d_320.laion2B-s29B-b131K-ft-soup/open_clip_pytorch_model.bin \
    --pretrain_mm_mlp_adapter models/osprey-v1.0-mlp2x-512px-convnext-pretrain-vicuna-7b-v1.5/mm_projector.bin \
    --mm_projector_type mlp2x_gelu \
    --mm_vision_select_layer -2 \
    --mm_use_im_start_end False \
    --mm_use_im_patch_token False \
    --image_aspect_ratio pad \
    --group_by_modality_length True \
    --bf16 True \
    --output_dir './exp/stage3_test' \
    --num_train_epochs 2 \
    --per_device_train_batch_size 1\
    --per_device_eval_batch_size 4 \
    --gradient_accumulation_steps 1 \
    --evaluation_strategy "no" \
    --save_strategy "steps" \
    --save_steps 2000 \
    --save_total_limit 1 \
    --learning_rate 2e-5 \
    --weight_decay 0. \
    --warmup_ratio 0.03 \
    --lr_scheduler_type "cosine" \
    --logging_steps 1 \
    --tf32 True \
    --model_max_length 2048 \
    --gradient_checkpointing True \
    --dataloader_num_workers 4 \
    --lazy_preprocess True \
    --report_to "none" \
    --group_by_modality_length False
```

**Comment by CircleRadon** [comment]
Hi, @jamespark3922. It seems that there are something wrong with the weights you load.
We have uploaded the stage2 weights on [https://huggingface.co/sunshine-lwt/Osprey-7b-stage2](https://huggingface.co/sunshine-lwt/Osprey-7b-stage2/tree/main), please feel free to have a try. After downloading the weights, please change the "mm_vision_tower" in `config.json` to the Absolute Path of open_clip_pytorch_model.bin.
If you encounter any further issues, kindly reach out to us here.
